### PR TITLE
Docs update 2.5.0 stabilization

### PR DESCRIPTION
**Pull request by hferrone**
## Changes
- Fixes: # .

### CODE REVIEW

**Review by davidkline-ms — APPROVED** [review]
overall, looks good. approved pending the resolution of the offline discussion of the Getting Started page

File: Documentation/Installation.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+title: Installation Guide
+description: Guide for installing MRTK-Unity in a new project.
+author: hferrone
+ms.author: v-hferrone
+ms.date: 09/9/2020
+ms.localizationpriority: high
+keywords: Unity,HoloLens, HoloLens 2, Mixed Reality, development, MRTK,
+---
+
+# Installation Guide
```
Comment (keveleigh) [inline Documentation/Installation.md:11]: One thing we've been using is ["Sentence case" for headers](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#capitalization).

Comment (keveleigh) [reply Documentation/Installation.md:11]:
```suggestion
# Installation guide
```

File: Documentation/WelcomeToMRTK.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+The Mixed Reality Toolkit (MRTK) is a cross-platform toolkit for building Mixed Reality experiences for Virtual Reality (VR) and Augmented Reality (AR). 
+
+## Development 
+
+If you're new to MRTK or Mixed Reality development in Unity, we recommend you start at the beginning of our [Unity development journey](https://docs.microsoft.com/windows/mixed-reality/unity-development-overview?tabs=mrtk%2Chl2). The Unity development journey is the **recommended starting point for MRTK**, specifically created to walk you through the installation, core concepts, and usage of MRTK in Unity. 
+
+> [!CAUTION]
+> The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.3.15**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
+
+## Documentation Versioning
```
Comment (keveleigh) [inline Documentation/WelcomeToMRTK.md:24]:
```suggestion
## Documentation versioning
```

File: Documentation/WelcomeToMRTK.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+![MRTK Logo](../Documentation/Images/MRTK_Logo_Rev.png)
+
+The Mixed Reality Toolkit (MRTK) is a cross-platform toolkit for building Mixed Reality experiences for Virtual Reality (VR) and Augmented Reality (AR). 
+
+## Development 
+
+If you're new to MRTK or Mixed Reality development in Unity, we recommend you start at the beginning of our [Unity development journey](https://docs.microsoft.com/windows/mixed-reality/unity-development-overview?tabs=mrtk%2Chl2). The Unity development journey is the **recommended starting point for MRTK**, specifically created to walk you through the installation, core concepts, and usage of MRTK in Unity. 
+
+> [!CAUTION]
+> The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.3.15**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
```
Comment (keveleigh) [inline Documentation/WelcomeToMRTK.md:22]: We should probably recommend 2019.4 over 2019.3.

```suggestion
> The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
```

Comment (keveleigh) [inline Documentation/WelcomeToMRTK.md:22]: We should probably recommend Unity 2019.4 over 2019.3.

```suggestion
> The Unity development journey currently uses **MRTK version 2.4.0** and **Unity 2019.4**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
```

Comment (keveleigh) [reply Documentation/WelcomeToMRTK.md:22]: Do we also want to reference 2.5?

```suggestion
> The Unity development journey currently uses **MRTK version 2.5.0** and **Unity 2019.4**. If you're working with other configurations it's still recommended that you start there, but you can also refer to the [installation instructions](Installation.md).
```

**Comment by keveleigh** [comment]
Looks like there are a couple broken links:

```
Build succeeded with warning.
[20-09-12 12:14:46.581]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.TocDocumentProcessor.Save](../../toc.yml#L0)Invalid file link:(~/../../Documentation/GettingStartedWithTheMRTK.md).
[20-09-12 12:14:46.581]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.TocDocumentProcessor.Save](../../toc.yml#L0)Invalid file link:(~/../../Documentation/GettingStartedWithTheMRTK.md).
[20-09-12 12:14:46.581]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.ConceptualDocumentProcessor.Save](../../Documentation/Tools/HolographicRemoting.md#L82)Invalid file link:(~/../../Documentation/GettingStartedWithTheMRTK.md#getting-help).
[20-09-12 12:14:46.581]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.ConceptualDocumentProcessor.Save](../../Documentation/Tools/HolographicRemoting.md#L109)Invalid file link:(~/../../Documentation/GettingStartedWithTheMRTK.md#getting-help).
[20-09-12 12:14:46.581]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.ConceptualDocumentProcessor.Save](../../Documentation/GettingStartedWithMRTKAndXRSDK.md#L7)Invalid file link:(~/../../Documentation/GettingStartedWithTheMRTK.md).
```

**Comment by davidkline-ms** [comment]
@hferrone, can you resolve the merge conflict?

**Comment by hferrone** [comment]
@davidkline-ms That should do it 👍

**Comment by keveleigh** [comment]
Looks like there's a .DS_Store file being committed. Is that needed?

**Review by keveleigh — CHANGES_REQUESTED** [review]
Is the .DS_Store file necessary? We haven't had one in the past

**Review by keveleigh — APPROVED** [review]

**Pull request merged**